go_modules/lib/dependabot/go_modules/update_checker.rb

```diff
@@ -41,6 +41,16 @@ def latest_version
         latest_resolvable_version
       end
 
+      def lowest_resolvable_security_fix_version
```

Comment:
We should add some test coverage for this method as well, is `resolvable?` available in the go update checker? I thought it wasn't

Reply:
You're correct, looks like I jumped the gun on this PR.
`lowest_security_fix_version` is also not available to UpdateChecker.